packages/orchestrator-ui-components/src/types/types.ts

```diff
@@ -63,6 +63,15 @@ export enum ProductLifecycleStatus {
     END_OF_LIFE = 'end of life',
 }
 
+export enum BadgeType {
```

Comment:
Some of these are multiple words. Therefore use _ in the names: like PRODUCT_TAG